cgi-bin/npg_qc

```diff
@@ -1,18 +1,20 @@
 #!/usr/local/bin/perl -T
+use strict;
+use warnings;
```

Comment:
There might be a reason why these two imports are not at the beginning of the file. This probably was important for the version of perl on the webteam's machines. As far as I remember, something to do with the warning module version.